modules/kubecfg/Makefile

```diff
@@ -2,6 +2,10 @@
 
 ## Validate manifests
 kubecfg/validate:
+	@echo "kubecfg/validate"
+	@echo "-------------------------------------------"
 	@@for env in $(shell ls ./rendered/environments) ; do \
```

Comment:
this is very specific BTW ... would be hard to validate anything that is not in "rendered/environments" ...

Reply:
both comments apply to all Makefiles below

Reply:
I could make it a variable, but I would prefer to make it the default as well

Reply:
it can still be a default and be a variable. This is not a default this is Hardcoded.

The more Hardcoding the less generic the less usage you will get

Reply:
Yeah, i will make it a var.

Reply:
Where possible i've used a var now, and in addition where the app supports it, I will just test against the parent directory and iet it recurse itself.  - but to be honest this can't be re-used in many places since there is a current requirement of `$$env/manifests`.

Let's worry about this if we need to re-use it, at which point maybe the same structure is already in place?